api/src/test/java/org/openmrs/module/bedmanagement/BedManagementDaoImplTest.java

```diff
@@ -0,0 +1,216 @@
+package org.openmrs.module.bedmanagement;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Encounter;
+import org.openmrs.EncounterType;
+import org.openmrs.Location;
+import org.openmrs.Patient;
+import org.openmrs.Visit;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.bedmanagement.dao.BedManagementDao;
+import org.openmrs.module.bedmanagement.entity.Bed;
+import org.openmrs.module.bedmanagement.entity.BedPatientAssignment;
+import org.openmrs.module.bedmanagement.entity.BedType;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class BedManagementDaoImplTest extends BaseModuleContextSensitiveTest {
+
+    private BedManagementDao bedManagementDao;
+    private Location defaultLocation;
+    private EncounterType defaultEncounterType;
+
+    @Before
+    public void setup() throws Exception {
+        executeDataSet("org/openmrs/include/standardTestDataset.xml");
+
+        bedManagementDao = Context.getRegisteredComponent("bedManagementDao", BedManagementDao.class);
+        assertNotNull("bedManagementDao should be initialized", bedManagementDao);
+
+        defaultLocation = Context.getLocationService()
+                .getLocationByUuid("8d6c993e-c2cc-11de-8d13-0010c6dffd0f");
+        assertNotNull("Default location should not be null", defaultLocation);
```

Comment:
I do not think we need this assertion because we are sure that we have this location in the test dataset.